container/container_test.go

```diff
@@ -140,19 +132,20 @@ func TestExistentContainerStatus(t *testing.T) {
 	containerJSONData := types.ContainerJSON{
 		ContainerJSONBase: &types.ContainerJSONBase{
 			ID:    containerID,
-			State: &types.ContainerState{},
+			State: &types.ContainerState{Running: true},
 		},
 	}
 
 	dt := dockertest.New()
 	c, _ := New(ClientOption(dt.Client()))
 
+	dt.ProvideServiceInspectWithRaw(swarm.Service{}, nil, nil)
 	dt.ProvideContainerList(containerData, nil)
 	dt.ProvideContainerInspect(containerJSONData, nil)
 
 	status, err := c.Status(namespace)
 	require.Nil(t, err)
-	require.Equal(t, STOPPED, status)
+	require.Equal(t, RUNNING, status)
```

Comment:
maybe this one should test _STARTING_ state, it kinda equivalent of _STOPPED_ in the previous implementation? it would be good if we test all the branches in the implementation of _Status()_ with different tests.